schemas/container.json

```diff
@@ -149,6 +149,13 @@
           "settings": {
             "type": "object"
           },
+          "filePaths": {
+            "type": "array",
```

Comment:
I would add here also: "minLength": 1, if there are no situation when you should receive an empty string.